gulpfile.esm.js

```diff
@@ -62,7 +62,7 @@ function runCommand(command) {
 //////////////////
 //////////////////
 function buildWebApp() {
-  return runCommand(`npm run action src/www/build_cordova -- --buildMode=${buildMode}`);
+  return runCommand(`npm run action src/www/build "${platform}" -- --buildMode=${buildMode}`);
```

Comment:
do you feel it makes sense to switch these all out with `run_action` so we don't have to use the`--`? btw i think in this case `platform` has to come after the `--`

Reply:
If you call run_action directly it's not in the `npm run` context, so the npm binaries are not in the PATH. We assume they are.

I don't need the `--` for positional args:
```
$ npm run action src/www/build ios -- --buildMode=release

> action
> bash ./scripts/run_action.sh "src/www/build" "ios" "--buildMode=release"

[Running src/www/build]
+ PLATFORM=ios
+ BUILD_MODE=debug
+ for i in '"$@"'
+ case $i in
+ for i in '"$@"'
+ case $i in
+ BUILD_MODE=release
```

Perhaps we should switch to positional arguments for the buildMode. But I'd rather explore that in another PR.

Reply:
Ah i'm thinking about passing them through to gulp, sorry

Reply:
I do worry making buildMode a positional argument will make it harder to have other dimensions e.g. `full`/`demo`

Reply:
If you're worried about the `--` perhaps a wrapper will help so we can do

`./outline <arguments>`

that just calls `npm run action -- "$@"`